### PR TITLE
Project: Advanced Building Blocks - Bubble Sort

### PR DESCRIPTION
**Pull request by ozovalihasan**
Project is done by following the instructions:

- bubble_sort and bubble_sort_by are used as method. bubble_sort is used to sort integers and bubble_sort_by for arrays containing integers
- It is checked by using rubocop

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hi Team 👋

You  seem to have done quite well on this @ozovalihasan  and @Daniduran-dev  👍

- [ ] - However, on testing out your code, it seems to fail at every point. Kindly rework it again, and make sure it passes with some examples before requesting another review.

Kindly fix this issue and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

**Comment by ozovalihasan** [comment]
- Problem is solved 
- One more optimization is added for both methods
- Fix all errors found by rubocop

File: README.md [hunk 1]
```diff
@@ -1 +1,62 @@
...
+## Authors
+
+👤 **Hasan Özovalı**
+
+- Github: [@ozovalihasan](https://github.com/ozovalihasan)
+- Twitter: [@ozovalihasan](https://twitter.com/ozovalihasan)
+- Linkedin: [Hasan Özovalı](https://www.linkedin.com/in/hasan-ozovali/)
+- Mail: [ozovalihasan@gmail.com](ozovalihasan@gmail.com)
+
+👤 **Hasan Özovalı**
```
Comment (euqueme) [inline README.md]: - [x] Change this name to Daniel Duran before you merge

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull request under the review by Maria Eugenia Queme P. (TSE)

### Hello @ozovalihasan and @Daniduran-dev  :balloon: :lollipop: 

### Thank you for doing the changes requested by the previous reviewer, your code is working just fine and you've used ruby convention very well so congratulations you may merge this branch :tada: :tada: :tada: 

#### Consideration:

- [ ] This is a collaborative project so there should be an even number of commit messages from both students, take this into account for future projects.

### Keep up the good work :clinking_glasses: 

PROJECT STATUS: APPROVED :green_apple:

**Comment by ozovalihasan** [comment]
Thanks for your feedback, @euqueme. I fixed the author's names. We will balance the number of commits done by each one for future projects

**Pull request merged**